R/system.R

```diff
@@ -705,6 +705,50 @@ getDBConnection <- function(type, host = NULL, port = "", databaseName = "", use
         connection_pool[[key]] <- conn
       }
     }
+  } else if (type == "mssqlserver") {# The type sqlserver is already used for RODBC based one and "mssqlserver" is passed from Exploratory Desktop.
+
+    # if platform is Linux use predefined one
```

Comment:
Please add comment on the reason of this.

Reply:
Added comment.